Lombiq.UIKit/Views/UiKit-Editor-Checkbox.cshtml

```diff
@@ -1,64 +1,61 @@
-@using Microsoft.AspNetCore.Mvc.Localization
 @using static Lombiq.UIKit.Constants.Classes
 
 @inject IHtmlGenerator HtmlGenerator
 
 @{
-    var customBlockClasses = Model.BlockClassName;
-    var blockId = string.IsNullOrEmpty(Model.BlockId) ? string.Empty : $"id={Model.BlockId}";
+    var viewModel = (UiKitEditorViewModel)Model.ViewModel;
```

Comment:
Why not `@model UiKitEditorViewModel`? In the other templates too.

Reply:
If you do that, you get a runtime error saying something like

>  The model item passed into the ViewDataDictionary is of type Shape but this ViewDataDictionary instance requires a model item of type UiKitEditorViewModel.

Reply:
Ah of course.